osu.Game/Overlays/SkinEditor/SkinBlueprintContainer.cs

```diff
@@ -39,7 +42,7 @@ protected override void LoadComplete()
             SelectedItems.BindTo(editor.SelectedComponents);
 
             var bindableList = new BindableList<ISerialisableDrawable> { BindTarget = targetContainer.Components };
-            bindableList.BindCollectionChanged(componentsChanged, true);
+            bindableList.BindCollectionChanged(componentsChanged, bindableList.Count != 0);
```

Comment:
To me it seems hacky to have an `Add` change event be called when there is literally nothing being added in the list. If needed it would probably be also alright to leave at `false`.